backend/app/tests/api/routes/test_stt_evaluation.py

```
@@ -1122,3 +1121,379 @@ def test_update_feedback_without_authentication(
+        assert isinstance(response_data["data"], list)
+
+    def test_list_audio_files_with_data(
+        self,
+        client: TestClient,
+        user_api_key_header: dict[str, str],
+        db: Session,
+        user_api_key: TestAuthContext,
+    ) -> None:
+        """Test listing audio files returns uploaded files."""
+        # Create test audio files
+        file1 = create_test_file(
```

Ruff (0.15.5)

[error] Local variable `file1` is assigned to but never used

Remove assignment to unused variable `file1`

(F841)